WordPress/src/main/java/org/wordpress/android/ui/mlp/BlockLayoutPreviewFragment.kt

```diff
@@ -27,7 +27,7 @@ class BlockLayoutPreviewFragment : LayoutPreviewFragment() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
-        chooseButton.setText(R.string.mlp_create_page)
+        (view.findViewById(R.id.chooseButton) as Button).setText(R.string.mlp_create_page)
```

Comment:
Suggestion (💡): This class, along with it sibling `DesignPreviewFragment`, who both share the same parent, the `LayoutPreviewFragment` class, can be further simplified and the parent class can be converted to `ViewBinding` as well.

I am not sure why these sibling classes end up duplicating all this code, but using an `abstract` pattern can simplify things a lot. Let's keep that in mind for when we will be converting this parent `LayoutPreviewFragment` class.

For example, with an appropriate refactor, the children classes, like the `DesignPreviewFragment` can become as simple as:
```
class DesignPreviewFragment : LayoutPreviewFragment() {
    companion object {
        const val DESIGN_PREVIEW_TAG = "DESIGN_PREVIEW_TAG"

        fun newInstance() = DesignPreviewFragment()
    }

    override fun getChooseButtonText() = R.string.hpp_choose_button

    override fun getViewModel() =
            ViewModelProvider(requireActivity(), viewModelFactory).get(HomePagePickerViewModel::class.java)
}
```

Reply:
@ParaskP7 - My goal was to remove the synthetic imports and limit the amount of refactoring to reduce risk of breaking other areas of the code. However, I did not notice that `BlockLayoutPreviewFragment` and `DesignPreviewFragment` both had the same parent, so in this case the refactor makes sense. That's why this is a draft. Thanks. :)

Reply:
Actually, in this case, I am thinking that this refactoring might be better off happening on a separate PR. I just mentioned that to increase our awareness on those classes and hierarchy. 🙏

Reply:
Note: Leaving as is in the PR and we'll address the refactoring separately. Going to resolve this conversation.